pve-mod-gui-sensors.sh

```diff
@@ -800,17 +799,17 @@ function save_sensors_data {
 	if (command -v sensors &>/dev/null); then
 		# Save sensors output
 		local filepath="${DEBUG_SAVE_PATH}/${DEBUG_SAVE_FILENAME}"
-		echo "Sensors data will be saved in $filepath"
-		
+		msg "Sensors data will be saved in $filepath"
+
 		# Prompt user for confirmation
 		read -p "Do you wish to continue? (y/n): " choice
-		case "$choice" in 
-			y|Y )
-				sensors -j > "$filepath"
+		case "$choice" in
+			y | Y)
```

Comment:
Can you change this to [yY] ), which we use throughout?

Reply:
Done.